po/en_GB.po

```diff
@@ -119,7 +124,8 @@ msgstr "Next CP Colour"
 
 #: charview.c:140
 msgid "The color used to draw the \"next\" control point of an on-curve point"
-msgstr "The colour used to draw the \"next\" control point of an on-curve point"
+msgstr ""
+"The colour used to draw the \"next\" control point of an on-curve point"
```

Comment:
What's going on here? How should a reviewer approach these changes?

Reply:
Yeah, it's not great, I'll play around with the config options to see if this can be improved, but the diff is always going to be huge, at least because of all the location/line number comments.

At a guess, I think it's running into some line wrapping limit and bumping it to the next line. I could be wrong though.

In terms of verifying the change, setting the env variable e.g. 'LANGUAGE=fr' and seeing that it still translates is one approach.

As for why I want to do this - clear out outdated translation strings, and to add new ones (untranslated) that have been added over time.

Reply:
Oh, I don't mean to question the intent. But short of trusting the tools I'm not sure how to review.

Reply:
It's still difficult to review, because they haven't really been kept up to date with the source changes, but try having a look at https://gist.github.com/jtanx/488c5c0ad57a2e73d239c16601cdb60c for a more manageable diff; this was done by following the steps in the second answer to https://stackoverflow.com/questions/2006351/gettext-po-files-under-version-control

Reply:
This was helpful. It's not like I read through every line, but I looked at enough to see that the overall content seems sound and about as expected, and to recognize some of the strings from the hotkey fix.